README.md

```diff
@@ -100,7 +112,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+In SQL a schema is a blueprint of the tables and attributes of the database based on which the database is created. The relationship is built and shown through joining tables on the foreign key. For example in the case of people and wishes (one to many), each wish has a person_id attribute that is a foreign key and can connect many wishes to one person.
```

Comment:
:+1:  Nice! Great job noting that the schema describes the database rather than a table!